packages/updatePrescriptionStatus/tsconfig.json

```diff
@@ -5,6 +5,6 @@
     "outDir": "lib"
   },
   "references": [],
-  "include": ["src/**/*", "tests/**/*"],
+  "include": ["src/**/*", "tests/**/*", "tests/tasks/*.json"],
```

Comment:
surely this is already covered by the entry before

Reply:
You'd think so, but `make compile` fails without it.